<launch>
    <param name="/behavior" value="light_follower" />

    <param name="/max_advance" value="0.15" />
    <param name="/max_turn_angle" value="0.7854" />
    <param name="/threshold_follower" value="7.1" />
    <param name="/run_algorithm" value="false" />

    <!-- <param name="/arena_hardware/light_bulbs" value="[False, False]" /> -->
    <include file="$(find mobile_base)/launch/mobile_base.launch" />
    
    <node name="motion_planner_node" pkg="motion_planner" type="motion_planner_node" output="screen" />
</launch>